Kinder/Data_files/Users.txt

```diff
@@ -1,2 +1,4 @@
 0 admin $2a$11$q3t/B9RNoQrwvM55v.drX.lYwXXLOv98dVjRTBPeDPLTJgtlaOLFK admin@gmail.com +12345678910 admin admin 0 2021-11-09
-1 admin2 $2a$11$jyUbGbbhsRAm7YIi9yZ4ceXclkjJt8XX4QwCm1mQXaBA3lSCdrgJm admin2@gmail.com +12345678911 admin2 admin2 0 2021-11-09
\ No newline at end of file
+1 admin2 $2a$11$jyUbGbbhsRAm7YIi9yZ4ceXclkjJt8XX4QwCm1mQXaBA3lSCdrgJm admin2@gmail.com +12345678911 admin2 admin2 0 2021-11-09
```

Comment:
Maybe you can write here passwords of these new accounts?)

Reply:
put them here: https://github.com/rokash7/Kinder#test-users-passwords